util/MySqlMigrations/Migrations/DatabaseContextModelSnapshot.cs

```diff
@@ -488,7 +489,8 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                 {
                     b.Property<int>("Id")
                         .ValueGeneratedOnAdd()
-                        .HasColumnType("int");
+                        .HasColumnType("int")
+                        .HasAnnotation("MySql:ValueGenerationStrategy", MySqlValueGenerationStrategy.IdentityColumn);
```

Comment:
ℹ️ Another override. The generated code put a Postgres annotation here instead which doesn't make sense. It also did it for SQLite.